provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/ProviderSpecialtiesResponse.java

```diff
@@ -14,12 +14,17 @@
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 @AllArgsConstructor(access = AccessLevel.PRIVATE)
 @JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
-public final class PpmsProviderSpecialtiesResponse {
+public final class ProviderSpecialtiesResponse {
   @JsonProperty("@odata.context")
   private String odataContext;
 
   private List<Value> value;
 
+  /** Lazy getter. */
+  public List<Value> getValue() {
+    return value;
```

Comment:
Good catch adding this, however it's not actually doing the check to initialize `value`

Reply:
File is updated... this preview doesn't show the resolved for some reason =/